CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Changelog
 
+## 6.0.0
+* Drop Puppet 3 support
+* New or changed parameters:
+    * Add `$autosignfile` for 1.16+ puppetca proxies and `$use_autosignfile` to be able to still use this module
+      with older proxy versions.
+* New or changed parameters on smart proxy plugin classes:
+    * Add `$ssl_disabled_ciphers` to the foreman_proxy::plugin::dynflow class.
+    * Add `$collect_status` to the foreman_proxy::plugin::monitoring class.
+* Other changes and fixes:
+    * Set foreman_smartproxy features for built-in and plugin modules.
```

Comment:
I added a sentence there, but would be grateful for better wordings, if wanted :)